ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/workers-rhel/provision/cucushift-installer-rehearse-azure-ipi-workers-rhel-provision-chain.yaml

```diff
@@ -0,0 +1,37 @@
+chain:
+  as: cucushift-installer-rehearse-azure-ipi-workers-rhel-provision
+  steps:
+  - ref: azure-provision-resourcegroup
+  - ref: azure-provision-vnet
+  - chain: azure-provision-bastionhost
+  - ref: ipi-conf
+  - ref: ipi-conf-azure
+  - ref: ipi-conf-azure-provisionednetwork
+  - ref: ipi-install-monitoringpvc
+  - ref: ipi-install-rbac
+  - ref: openshift-cluster-bot-rbac
+  - ref: ipi-install-install
+  - ref: ipi-install-times-collection
+  - ref: workers-rhel-azure-provision
+  - chain: workers-rhel
+  env:
+  - name: RHEL_WORKER_COUNT
+    default: 2
+    documentation: The number of rhel worker to be provisioned.
+  - name: RHEL_IMAGE
+    default: "8.5.2021121504"
+    documentation: |-
+      The name of rhel vm/template used to clone RHEL instances
+  - name: RHEL_IMAGE_SKU
+    default: "8-lvm-gen2"
+    documentation: |-
+      The name of rhel image SKU used to clone RHEL instances
+  - name: PLATFORM_VERSION
```

Comment:
The same comments as ci-operator/step-registry/workers-rhel/azure-provision/workers-rhel-azure-provision-ref.yaml

Reply:
I guess we can drop this env var now.

Reply:
Ah, I missed that, you are right, let us keep it here.